charts/metrics-agent/Chart.yaml

```diff
@@ -14,8 +14,8 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 2.14.1
+version: 2.14.2
```

Comment:
We do 1:1 versioning between the agent and the chart, so need to update version.go and the image referenced in the chart